notebooks/hyp_search.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
add example where i save the results to a csv every iteration and then load the best model from the csv file directly